model/tensorflow_hub/examples/tfhub_text_classifier/predict.sh

```diff
@@ -0,0 +1,11 @@
+dffml predict all \
+  -model text_classifier \
+  -model-predict sentiment:int:1 \
+  -model-classifications 0 1 \
```

Comment:
The issue is because we have a different number of classifications, when we go to do the accuracy command it's loading a different model.

Reply:
https://github.com/intel/dffml/blob/1635ed60627139f3cb50d6a8e5fbb3fcd678d1e4/model/tensorflow_hub/dffml_model_tensorflow_hub/text_classifier.py#L119-L124

Reply:
Ah, damn. I was playing with the model wondering why do we not have the 3 classifications there and I let that slip :facepalm:

Reply:
This happens to me ALL THE TIME I sometimes wonder if we should ditch the hashing for a more explicit way. Let's bring it up in the sync meeting tomorrow